perf-common/src/main/java/com/github/projectflink/common/AnalyzeTool.java

```diff
@@ -33,7 +33,7 @@ public Result(DescriptiveStatistics latencies, SummaryStatistics throughputs, Ma
 		}
 	}
 
-	public static Result analyze(String file, List<String> toIgnore) throws IOException {
+	public static Result analyze(String file) throws IOException {
```

Comment:
please then also remove the commented-out code at the bottom of the class that was using this field